### PR TITLE
Phishing Warning

### PR DESCRIPTION
**Pull request by mgurgel**
**Reviewer:** @shakyShane 

https://app.asana.com/0/1206594217596623/1207336102226042/f

## Description:

Phishing warning state for Privacy Dashboard

## Steps to test this PR:

1. Run `npm run preview`
2. Select macOS
3. Select `phishing` from the dropdown
4. Confirm that phishing warning is shown

### CODE REVIEW

File: shared/locales/en/site.json [hunk 1]
```diff
@@ -131,6 +131,14 @@
         "title": "Connection May Be Insecure",
         "note": "Shown as the button text when the connection is using an invalid certificate."
     },
+    "phishingWebsite": {
+        "title": "Site May Be Deceptive",
+        "note": "Shown as the button text when the URL is reported as a phishing website."
+    },
+    "phishingWebsiteDesc": {
```
Comment (shakyShane) [inline shared/locales/en/site.json:138]: Are these strings confirmed yet?

File: shared/js/ui/platform-features.mjs [hunk 2]
```diff
@@ -40,6 +40,7 @@ export function createPlatformFeatures(platform) {
         supportsHover: desktop.includes(platform.name),
         initialScreen: screen,
         supportsInvalidCerts: platform.name !== 'browser' && platform.name !== 'windows',
+        supportsPhishingWarning: platform.name === 'macos', // TODO: Add a version number?
```
Comment (shakyShane) [inline shared/js/ui/platform-features.mjs:43]: This 'todo' comment, is it because Thom's work is not being released to all macOS versions?

**Review by shakyShane — COMMENTED** [review]
Looking great! just a couple of questions :)

File: shared/js/ui/platform-features.mjs (hunk 2 above)
Comment (mgurgel) [reply shared/js/ui/platform-features.mjs:43]: No, I just wanted to check whether we need to lock it to a minimum version. But now that I think about it, previous versions wouldn’t carry this version of the C-S-S, so my question doesn’t really make sense.

File: shared/locales/en/site.json (hunk 1 above)
Comment (mgurgel) [reply shared/locales/en/site.json:138]: Yes, according this https://app.asana.com/0/38424471409662/1207197456211299/f but they might get changed during Ship Review

File: shared/js/ui/platform-features.mjs (hunk 2 above)
Comment (mgurgel) [reply shared/js/ui/platform-features.mjs]: Comment removed!

**Review by shakyShane — APPROVED** [review]
@mgurgel - approving for now, I assume any changes in the ship review will be copy/icons only. We can leave this open whilst that process goes on :)

**Comment by jonathanKingston** [comment]
Heads up you can take the merge commit from: https://github.com/duckduckgo/privacy-dashboard/pull/182 (assuming it looks correct).

**Comment by mgurgel** [comment]
Closing in favor of https://github.com/duckduckgo/privacy-dashboard/pull/182

**Pull request closed without merging**